CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 2.1.1+0
+- Fixes [issue](https://github.com/alippo-com/GrowthBook-SDK-Flutter/issues/47)
```

Comment:
As it updates the Dio version to the major we should consider updating the SDK version to the major. 
```md 
# 3.0.0+0
```

Reply:
Done.